crud/src/main/java/com/redhat/lightblue/crud/mongo/MongoCRUDController.java

```diff
@@ -349,11 +347,10 @@ public CRUDFindResponse find(CRUDOperationContext ctx,
                 ctx.addError(Error.get(MongoCrudConstants.ERR_NO_ACCESS, "find:" + ctx.getEntityName()));
             }
         } catch (Error e) {
-            // rethrow lightblue error
-            throw e;
+            ctx.addError(e);
```

Comment:
Are Error.get always called when an error happens? If so and they are all supposed to log, then statements
   LOGGER.error(e.getMessage(), e);
in catch are redundant.